README.md

```diff
@@ -51,4 +51,4 @@ Another typical change would be the integration of your company-wide monitoring
 * [Tips and tricks](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/docs/tips-and-tricks.md) for working with Kubernetes and Alfresco Content Services.
 * The images downloaded directly from Docker Hub, or Quay.io are for a limited trial of the Enterprise version of Alfresco Content Services that goes into read-only mode after 2 days. Request an extended 30-day trial at
  https://www.alfresco.com/platform/content-services-ecm/trial/docker
-* You'll need your [Quay.io](https://quay.io/) account credentials to access the Docker images. If you don't already have these credentials, contact [Alfresco Support](https://support.alfresco.com/).
+* Alfresco customers can request Quay.io credentials by logging a ticket with [Alfresco Support](https://support.alfresco.com/). These credentials are required to download private (Enterprise-only) Docker images from Quay.io.
```

Comment:
Probably better to use "pull" instead of "download".

Reply:
Done